app/views/new_document/guidance.govspeak.md

```diff
@@ -50,7 +50,7 @@ The [Parliament website](https://www.parliament.uk/) covers the work of parliame
 
 ### Legislation.gov.uk
 
-Bills which have been approved by the Commons, the Lords, and The Queen become law and are recorded on [Legislation.gov.uk](http://www.legislation.gov.uk/).
+Bills which have been approved by the Commons, the Lords, and The King become law and are recorded on [Legislation.gov.uk](http://www.legislation.gov.uk/).
```

Comment:
Minor, we should change this to "the King" as per the decisions made in: https://docs.google.com/document/d/1VXFaCkM3cz9vqLCeVK9UV6DKnGZ5VyX0qclND9CLjvw/edit?disco=AAAAgjPvmMo